src/app/views/query-runner/request/headers/RequestHeaders.tsx

```diff
@@ -66,6 +69,7 @@ class RequestHeaders extends Component<IRequestHeadersProps, any> {
       this.setState({
         headerName: '',
         headerValue: '',
+        announcedMessage: translateMessage('Request Header added')
```

Comment:
Why does the announcedmessage need to go into the state?

Reply:
I found it to be an easier way to change what's announced based on whether a user adds/deletes an item